starfish/imagestack/physical_coordinate_calculator.py

```diff
@@ -42,8 +44,8 @@ def calc_new_physical_coords_array(physical_coordinates: xr.DataArray,
     return new_coords
 
 
-def _needs_coords_recalculating(x_indexers: Union[int, slice], y_indexers: Union[int, slice]
-                                ) -> bool:
+def _needs_coords_recalculating(
+        x_indexers: Union[int, slice], y_indexers: Union[int, slice]) -> bool:
     if isinstance(x_indexers, int) or isinstance(y_indexers, int):
```

Comment:
If the return definition goes over the line limit you'd prefer to just have everything on the next line?

Reply:
Yup.  The rationale is the same: it minimizes LOC churn if we rename methods.